financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentViewModel.kt

```diff
@@ -76,7 +74,6 @@ internal class ConsentViewModel @Inject constructor(
     fun onContinueClick() {
         suspend {
             eventTracker.track(ConsentAgree)
-            FinancialConnections.emitEvent(Name.CONSENT_ACQUIRED)
```

Comment:
is this expected?

Reply:
Correct - the integration branch was showcasing how an event can be emitted on the flow. 

However, this (and all the other) events now come in backend responses, so clients don't have to manually trigger them.